examples.js

```diff
@@ -0,0 +1,328 @@
+            filename: 'document.pdf'  // Required for Buffers don't have filenames
+        });
+        console.log('Buffer conversion:', result2);
+        
+    } catch (error) {
+        console.error('Error:', error);
+    }
+}
+
+/**
+ * Example 9: Fire-and-forget with callback URL
+ * Demonstrates async pattern where client returns immediately and server notifies a webhook
+ * Useful for long-running conversions or when processing in a queue system
+ */
+async function fireAndForgetConversion() {
+    const client = new IDRCloudClient(`https://cloud.idrsolutions.com/cloud/${CONVERTER.BUILDVU}`, {
+        token: 'your-api-token'
+    });
+    
+    try {
+        // Client returns immediately after upload, server notifies callbackUrl when done
+        const response = await client.convert(
+            {
+                input: INPUT.UPLOAD,
+                file: './example.pdf',
+                callbackUrl: 'https://example.com/webhook'  // Server will POST result here
+            },
+            {
+                poll: false  // Explicit: don't wait for completion
+            }
+        );
+        
+        console.log('Conversion started:', response.uuid);
+        console.log('Server will notify webhook when complete');
+        
+    } catch (error) {
+        console.error('Error starting conversion:', error);
+    }
+}
+
+/**
+ * Example 10: Poll with progress AND callback URL
+ * Demonstrates using both client-side polling and server-side webhook notification
+ * Useful when you want immediate feedback AND persistent notification
+ */
+async function pollWithCallback() {
+    const client = new IDRCloudClient(`https://cloud.idrsolutions.com/cloud/${CONVERTER.BUILDVU}`, {
+        token: 'your-api-token'
+    });
+    
+    try {
+        // Client polls for completion AND server notifies webhook
+        const result = await client.convert(
+            {
+                input: INPUT.UPLOAD,
+                file: './example.pdf',
+                callbackUrl: 'https://example.com/webhook'
+            },
+            {
+                poll: true,  // Client waits (this is the default)
+                onProgress: (progress) => {
+                    console.log('Progress:', progress.state);
+                }
+            }
+        );
+        
+        console.log('Conversion complete!', result);
+        console.log('Webhook also received notification');
+        
+    } catch (error) {
+        console.error('Error:', error);
+    }
+}
+
+/**
+ * Example 11: Using streams for large files
+ * Demonstrates converting from a stream by converting it to a Blob
+ * Useful for processing data on-the-fly or streaming from external sources (e.g. HTTP responses)
+ */
+async function streamConversion() {
+    const client = new IDRCloudClient(`https://cloud.idrsolutions.com/cloud/${CONVERTER.BUILDVU}`, {
+        token: 'your-api-token'
+    });
+    
+    try {
+        // Option 1: Using fs.createReadStream (Node.js)
```

Comment:
I don't see an option 2, so would suggest changing to:
```suggestion
        // Using fs.createReadStream (Node.js)
```